set.mm

```diff
@@ -15808,19 +15928,19 @@ definition has the properties we expect of proper substitution (see
 
      There are no restrictions on any of the variables, including what
      variables may occur in wff ` ph ` .  (Contributed by NM, 10-May-1993.) $)
-  df-sb $a |- ( [ y / x ] ph <->
+  df-sbOLD $a |- ( [ y / x ] ph <->
               ( ( x = y -> ph ) /\ E. x ( x = y /\ ph ) ) ) $.
```

Comment:
Would it be possible to prove this as a theorem (e.g. `dfsb2`), and then replace all instances of `df-sb` with `dfsb2`?
These would still use `ax-13`, but then it's possible to improve them later.

Reply:
I've used versions of this approach with good results in the past. For example in changing the definitions of `seq` and `sum_` in iset.mm (this was spread over many pull requests from 2022 to 2023).

I can see two variations:
1. Change the definition (as in this pull request), then prove `df-sbOLD` (or whatever we call it) and change existing usages, as @tirix suggests in the comment above
2. Don't change the definition yet, but introduce our desired new definition as a theorem (`dfsb7al` or whatever we'd call it) and slowly change usages from `df-sb` to the new one. At some later point we'd change it from a theorem to a definition.

What is in the pull request now is giving an error `Axiom df-sbOLD has failed the definitional soundness check. The previous axiom df-sb uses the symbol being defined in this axiom.` and I don't think we'd want to relax the definition check given the potential for introducing unsoundness if we missed anything about whether the definitions are equivalent.